### PR TITLE
update docs for 0.10.0-rc1

### PR DESCRIPTION
**Pull request by gwenshap**

### CODE REVIEW

File: 0100/api.html [hunk 1]
```diff
@@ -24,12 +24,12 @@
...
 	    &lt;groupId&gt;org.apache.kafka&lt;/groupId&gt;
 	    &lt;artifactId&gt;kafka-clients&lt;/artifactId&gt;
-	    &lt;version&gt;0.9.0.0&lt;/version&gt;
+	    &lt;version&gt;0.10.0.0&lt;/version&gt;
 	&lt;/dependency&gt;
 </pre>
 
 Examples showing how to use the producer are given in the
-<a href="http://kafka.apache.org/090/javadoc/index.html?org/apache/kafka/clients/producer/KafkaProducer.html" title="Kafka 0.9.0 Javadoc">javadocs</a>.
+<a href="http://kafka.apache.org/0100/javadoc/index.html?org/apache/kafka/clients/producer/KafkaProducer.html" title="Kafka 0.10.0 Javadoc">javadocs</a>.
```
Comment (granthenke) [inline 0100/api.html:32]: Just a reminder that javadocs need to be pushed before this will work.

Comment (gwenshap) [inline 0100/api.html:32]: Thanks!!!

**Comment by gwenshap** [comment]
Haha, change just went from 13 files to 786 :)

Thats the javadocs...

**Comment by ijuma** [comment]
It seems fine to me, but I only had a chance to look at the diff (i.e. I didn't test it).

**Pull request closed without merging**